client/components/rewind-credentials-form/index.jsx

```diff
@@ -171,40 +202,31 @@ export class CredentialsForm extends Component {
 				</div>
 
 				<FormFieldset>
-					<FormLabel htmlFor="wordpress-path">{ translate( 'Upload Path' ) }</FormLabel>
-					<FormTextInput
-						name="path"
-						id="wordpress-path"
-						placeholder="/public_html/wordpress-site/"
-						value={ get( this.state.form, 'path', '' ) }
-						onChange={ this.handleFieldChange }
-						disabled={ formIsSubmitting }
-						isError={ !! formErrors.path }
-					/>
-					{ formErrors.path && <FormInputValidation isError={ true } text={ formErrors.path } /> }
-				</FormFieldset>
-
-				<FormFieldset>
-					<FormLabel>{ translate( 'Private Key' ) }</FormLabel>
-					<Button disabled={ formIsSubmitting } onClick={ this.togglePrivateKeyField }>
-						{ showPrivateKeyField
-							? translate( 'Hide Private Key' )
-							: translate( 'Show Private Key' ) }
+					<Button disabled={ formIsSubmitting } onClick={ this.toggleAdvancedSettings }>
+						{ translate( 'Advanced settings' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Advanced Settings' )` *ES Score: 19*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/alr/creds-form-refactor)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).